developer_manual/design/navigation.rst

```diff
@@ -0,0 +1,153 @@
+.. sectionauthor:: John Molakvoæ <skjnldsv@protonmail.com>
+.. codeauthor:: John Molakvoæ <skjnldsv@protonmail.com>
+..  _navigation:
+
+===============
+App Navigation
+===============
+
+Introduction
+=============
+
+The main navigation menu represents the main navigation of your app.
+
+It needs to be:
+
+* Organised
+* Simple
+* Responsive
+
+Nextcloud provides a very organized way of building menus.
+We implemented various essential functions and provide easy way of using them.
+
+.. figure:: ../images/navigation.png
+   :alt: Navigation screenshot
+
+Basic layout
+=============
+
+.. code-block:: html
+
+	<div id="app-navigation">
+	    <ul>
+	        <li><a href="#">First level entry</a></li>
+	        <li>
+	            <a href="#">First level container</a>
+	            <ul>
+	                <li><a href="#">Second level entry</a></li>
+	                <li><a href="#">Second level entry</a></li>
+	            </ul>
+	        </li>
+	    </ul>
+	</div>
+
+Utils
+======
+
+Each entry is allowed to have a counter and/or a button for user interaction.
+
+* The ``app-navigation-entry-utils`` snippet need to be placed right next to the main link of your entry.
+* Maximum **two** items are allowed into the utils section. You can have:
+	* Two buttons
+	* One button and one counter
```

Comment:
could you like `counter` with the *Counter* section?

Reply:
Yes! Thx! There is still a lot to do :)